src/redux/books/books.js

```diff
@@ -1,20 +1,44 @@
-/* eslint-disable no-fallthrough */
+import { v4 as uuidv4 } from 'uuid';
 // Actions
 const Actions = [
   { ADD: 'bookstore/book/ADD' },
   { REMOVE: 'bookstore/book/REMOVE' },
 ];
+// state initialized as array
+
+const initialState = [
+  {
+    id: uuidv4(),
+    title: 'The First Game',
+    author: 'Suarez',
+    topic: 'Actions',
+    category: 'Economy',
+  },
+  {
+    id: uuidv4(),
+    title: 'The Second Science',
+    author: 'Daniel',
+    topic: '',
+    category: 'Science Fiction',
+  },
+  {
+    id: uuidv4(),
+    title: 'The First Game',
+    author: 'Suarez',
+    topic: 'Actions',
+    category: 'Action',
+  },
```

Comment:
- [ ] **[OPTIONAL]** Good job so far, your state is immutable in all reducers, and the List of books is displayed from the Redux store using `useSelector` function. However, I suggest you remove the data you added to the initial state because we expect to have only the books added by a user.